### ISSUE TITLE
Update phone numbers style guide entry to include TTY in aria labels

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#40611]
## Issue Description
Our style guide currently shows code for TTY numbers where "TTY" is not inside the aria labels. The accessibility team confirmed that it should be. The proper coding is: 

\<a href="tel:711" aria-label="TTY. 7 1 1.">711</a>

This needs to be updated in the [phone numbers entry](https://design.va.gov/content-style-guide/dates-and-numbers#phone-numbers) of the style guide.

These changes need to be done in Github.
The style guide in GH: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/tree/master/src/_content-style-guide

## Tasks
- [x] Update style guide entry (Randi)
- [x] Review PR and merge (Beth)

## Acceptance Criteria
- [x] Correct TTY code is live in the style guide

**Comment by RLHecht** [comment]
@megzehn Just an FYI that style guide updates are done in github rather than Drupal so the work is a little different. Happy to walk you through the way we make changes to the style guide so that you'll know for future updates.

**Comment by RLHecht** [comment]
PR for this was merged. Moving to validate and will close once it's live.

**Comment by RLHecht** [comment]
Work is complete. This may be updated again if the CMS can use the va-telephone component but that won't be until a later date. Closing ticket.
